administrator/components/com_fields/helpers/fields.php

```diff
@@ -170,8 +173,23 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 					{
 						try
 						{
+							/*
+							 *On before field render
+							 *Event allow plugins to modfify the output of the field before it is rendered
+							 */
+                            $dispatcher = JEventDispatcher::getInstance();
+							if($field->value)
+								$dispatcher->trigger('onFieldBeforeRender',array($context,'mustache',&$field));
```

Comment:
WHat I wanted to ask, is the $layoutFile/'mustache' parameter really needed? Would be nice to avoid it.